pkgs/amino/json_decode.go

```diff
@@ -413,45 +419,53 @@ func extractJSONTypeURL(bz []byte) (typeURL string, value json.RawMessage, err e
 	anyw := new(anyWrapper)
 	err = json.Unmarshal(bz, anyw)
 	if err != nil {
-		err = fmt.Errorf("cannot parse Any JSON wrapper: %v", err)
+		err = fmt.Errorf("cannot parse Any JSON wrapper: %w", err)
+
 		return
 	}
 
 	// Get typeURL.
 	if anyw.TypeURL == "" {
 		err = errors.New("JSON encoding of interfaces require non-empty @type field")
+
 		return
 	}
 	typeURL = anyw.TypeURL
 	value = anyw.Value
+
 	return
 }
 
 func deriveJSONObject(bz []byte, typeURL string) (res []byte, err error) {
 	str := string(bz)
 	if len(bz) == 0 {
 		err = errors.New("expected JSON object but was empty")
+
```

Comment:
what's this one?

Reply:
This is the [nlreturn](https://github.com/ssgreg/nlreturn) linter, that forces a newline before return and branch statements

Reply:
Can you remove it, for the same reason as above?

Reply:
I think it's a good call to remove it 👌 

Reverted the linter here:
[a04f258](https://github.com/gnolang/gno/pull/472/commits/a04f258a17a9ba95646f448e8854afcc515c84e5)

Reply:
thank you